Core/HLE/sceNet.h

```diff
@@ -186,11 +194,43 @@ struct ApctlHandler {
 	u32 argument;
 };
 
+struct ApctlArgs {
+	u32_le data[5]; // OldState, NewState, Event, Error, ArgsAddr
+};
+
 class PointerWrap;
 
+class AfterApctlMipsCall : public PSPAction {
+public:
+	AfterApctlMipsCall() {}
+	static PSPAction* Create() { return new AfterApctlMipsCall(); }
+	void DoState(PointerWrap& p) override {
+		auto s = p.Section("AfterApctlMipsCall", 1, 1);
+		if (!s)
```

Comment:
Something strange also happened here when i put breakpoint here.
This part of the code shouldn't be reached when loading an old savestate that have no `AfterApctlMipsCall` yet, right?
But apparently one of the game (Naruto Ultimate Ninja Heroes 3) i tested with was able to reach this breakpoint but having corrupted data (`s.ver` contains a negative value, and i also saw some text like "AfterMatchingMipsCall\4" witin `s`)
It might be related to the weird PSPAction that suppose to trigger `AfterApctlMipsCall::run` but running `AfterMatchingMipsCall::run` instead after loading from old savesate which doesn't have `AfterApctlMipsCall` yet.

Reply:
If it found some data it could accidentally interpret as a valid section header, we'd indeed be in trouble... I don't really know what's best to do here :/

Reply:
After rearranging PSPAction creation/registration it's no longer reach this line after loading from old save state